src/services/scriptingservice.cpp

```diff
@@ -2219,3 +2220,38 @@ void ScriptingService::onScriptThreadDone(ScriptThread *thread) {
         }
     }
 }
+
+/**
+ * Returns the default cache folder of QON
+ * @param {QString} subDir the subfolder to create and use
+ */
+QString ScriptingService::cacheDir(const QString &subDir) const {
+    QString cacheDir = QStandardPaths::writableLocation(QStandardPaths::CacheLocation) + QString("/scripts/");
+    if(!subDir.isEmpty()){
```

Comment:
Can you please use the same coding style like everywhere else (space before brackets).